sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionTestEnvironment.cs

```diff
@@ -7,18 +7,18 @@
 
 namespace Azure.Monitor.Ingestion.Tests
 {
-    public class IngestionClientTestEnvironment : TestEnvironment
+    public class MonitorIngestionTestEnvironment : TestEnvironment
     {
-        public string DCRImmutableId => GetRecordedVariable("AZURE_MONITOR_INGESTION_LOGS_DCR_RULE_ID");
+        public string DCRImmutableId => GetRecordedVariable("AZURE_MONITOR_INGESTION_LOGS_DCR_RULE_ID", options => options.IsSecret());
 
-        public string DCREndpoint => GetRecordedVariable("AZURE_MONITOR_INGESTION_DATA_COLLECTION_ENDPOINT");
+        public string DCREndpoint => GetRecordedVariable("AZURE_MONITOR_INGESTION_DATA_COLLECTION_ENDPOINT", options => options.IsSecret());
```

Comment:
I don't think we would consider this a secret, and this will probably break tests in playback as it would not be a URI.